src/test/js/shared/idam_helper.js

```diff
@@ -468,6 +468,29 @@ class IdamHelper extends Helper {
         });
     }
 
+    registerUserWithId(bearerToken, userEmail, userFirstName, userLastName, userId, userRoles) {
+        const data = {
+            id: userId,
+            email: userEmail,
+            firstName: userFirstName,
+            lastName: userLastName,
+            roles: [userRoles]
+        };
+
+        return fetch(`${TestData.IDAM_API}/api/v1/users/registration`, {
+            agent: agent,
+            method: 'POST',
+            body: JSON.stringify(data),
+            headers: {'Content-Type': 'application/json', 'Authorization': 'Bearer ' + bearerToken},
+        }).then((response) => {
+            if (response.status != 200) {
+                console.log('Error creating user', response.status);
+                console.log(JSON.stringify(data))
+                throw new Error()
+            }
+        });
```

Comment:
just pointing out there there's not catch block in here so any fetch related errors thrown won't be logged

Reply:
Thanks, it should be fine as the response status being logged to debug.